style.scss

```diff
@@ -0,0 +1,631 @@
+    font-family: var(--ff-body);
+    font-size: var(--fs-body);
+    font-weight: var(--fw-regular);
+    color: var(--clr-primary-900);
+}
+
+.container {
+    --max-width: 1110px;
+    --padding: 1rem;
+
+    width: min(var(--max-width), 100% - (var(--padding) * 2));
+    margin-inline: auto;
+}
+
+.header {
+    padding: var(--size-400) 0;
+}
+
+.portfolio__intro {
+    display: flex;
+    flex-direction: column;
+    padding-block: var(--size-900);
+    
+    .description {
+        padding: 0 0.8125rem;
+    }
+
+}
+
+.portfolio__skills {    
+    &-grid {
+        display: grid;
+        gap: var(--size-500);
+        grid-auto-rows: auto;
+        grid-template-columns: minmax(1rem, 1fr) minmax(1rem, 1fr);
+    }
+    
+    &-grid-item {
+        grid-column: 1 / -1;
+        
+        &--ui-ux {
+            grid-column: 1 / 2;
+        }
+
+        &--apps {
+            grid-column: 2 / 3;
+        }
+    }
+    
+    &-grid-text{
+        display: flex;
+        flex-direction: column;
+        justify-content: end;
+        text-align: start;
+        text-decoration: none;
+        padding: var(--size-500);
+        background-origin: content-box;
+        background-repeat: no-repeat;
+        background-position: top right;
+        color: var(--clr-neutral-100);
+        
+        
+        &--graphic-design {
+            background-image: url('/assets/pattern-graphic-design.svg');
+            min-width: 1rem;
+            min-height: 24.25rem;
+            background-color: var(--clr-primary-900-hover);
+        }
+        &--ui-ux {
+            background-image: url('/assets/pattern-ui-ux.svg');
+            min-width: 1rem;
+            min-height: 11.375rem;
+            background-color: var(--clr-accent-400-hover);
+        }
+        &--apps {
+            background-image: url('/assets/pattern-apps.svg');
+            min-width: 1rem;
+            min-height: 11.375rem;
+            background-color: var(--clr-accent-100);
+        }
+        &--illustrations {
+            background-image: url('/assets/pattern-illustrations.svg');
+            min-width: 1rem;
+            min-height: 11.375rem;
+            background-color: var(--clr-accent-400);
+        }
+        &--photography {
+            background-image: url('/assets/pattern-photography.svg');
+            min-width: 1rem;
+            min-height: 11.375rem;
+            background-color: var(--clr-accent-200);
+        }
+    
+        &--motion-graphics {
+            background-image: url('/assets/pattern-motion-graphics.svg');
+            min-width: 1rem;
+            min-height: 11.375rem;
+            background-color: var(--clr-accent-800);
+        }
+    }
+}
+
+.portfolio__about-me {
+    display: flex;
+    
+    &-intro {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+    }
+}
+
+.portfolio__carousel {
+    padding-block: var(--size-900);
+    text-align: center;
+    
+    &-slider {
```

Comment:
Move these as 1st level class, `.portfolio__carousel {...},` `.portfolio__carousel-slider {...}` to avoid unnecessary nesting.